src/parser/TranspileState.ts

```diff
@@ -68,13 +69,30 @@ export class TranspileState {
     /**
      * Shorthand for creating a new source node
      */
-    public sourceNode(locatable: { range: Range }, code: string) {
-        let result = new SourceNode(
-            locatable.range.start.line,
+    public sourceNode(locatable: { range: Range }, code: string | SourceNode | Array<string | SourceNode>): SourceNode | undefined {
+        return new SourceNode(
+            //convert 0-based range line to 1-based SourceNode line
+            locatable.range.start.line + 1,
```

Comment:
This is a bug that probably caused all sorts of issues when stepping through transpiled code in the debugger.